common/app/views/fragments/javaScriptLaterSteps.scala.html

```diff
@@ -334,6 +334,64 @@
 
                 });
 
+                @for(membershipAccess <- item.membershipAccess if conf.Switches.MembersAreaSwitch.isSwitchedOn) {
+                    /**
+                     * Membership access
+                     * Items with either of the following fields require Membership access
+                     * - membershipAccess=members-only
+                     * - membershipAccess=paid-members-only
+                     */
+                    (function(isModern) {
+
+                        function reveal() {
+                            $('body').removeClass('has-membership-access-requirement');
+                        }
+
+                        /**
+                         * Authenticating requires CORS and withCredentials
+                         * If we don't cut the mustard then pass through.
+                         */
+                        if(!isModern) {
+                            reveal();
```

Comment:
As discussed, we can't authenticate non-mustard browsers at all so pass through and display the content to them.